src/main/java/com/salesforce/mirus/metrics/ConnectorJmxReporter.java

```diff
@@ -8,8 +8,10 @@
 
 package com.salesforce.mirus.metrics;
 
+import com.google.common.collect.Sets;
```

Comment:
Is it necessary to take a dependency on the Google commons library here? Can you  just use a standard HashSet?